tests/integration/testdata/sync/Ruby/function/app.rb

```diff
@@ -0,0 +1,39 @@
+# require 'httparty'
+require 'json'
+require 'layer_test'
+
+def lambda_handler(event:, context:)
+  # Sample pure Lambda function
+
+  # Parameters
+  # ----------
+  # event: Hash, required
+  #     API Gateway Lambda Proxy Input Format
+  #     Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
+
+  # context: object, required
+  #     Lambda Context runtime methods and attributes
+  #     Context doc: https://docs.aws.amazon.com/lambda/latest/dg/ruby-context.html
+
+  # Returns
+  # ------
+  # API Gateway Lambda Proxy Output Format: dict
+  #     'statusCode' and 'body' are required
+  #     # api-gateway-simple-proxy-for-lambda-output-format
+  #     Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
+
+  # begin
+  #   response = HTTParty.get('http://checkip.amazonaws.com/')
+  # rescue HTTParty::Error => error
+  #   puts error.inspect
+  #   raise error
+  # end
```

Comment:
Similar to python one, let's uncomment these lines so we can test a function with dependency.